mqtt/mqtt3/README.md

```diff
@@ -20,6 +20,31 @@ The crate is not published to crates.io yet. Please generate docs locally with `
 
 See the `examples/` directory for examples of a publisher and subscriber, and for how to set a will.
 
+# Fuzz testing
+
+The crate is coming with fuzz tests. 
+
+## Prerequisites
+
+```bash
+sudo apt install tmux binutils
+cargo install afl
+```
+
+## Run fuzzer
+
+```bash
+echo 'core' | sudo tee /proc/sys/kernel/core_pattern
+
+which gold
+
+/build/linux/mqtt-fuzz.sh ../mqtt3-fuzz/
```

Comment:
This should be `build/linux/mqtt-fuzz.sh ` ? No leading `/`.

Also if you're going to run this from the `mqtt` directory, then the path to `mqtt3-fuzz` would be just `mqtt3-fuzz`, not `../mqtt3-fuzz`